examples/cliffordexamples/destabilizer.txt

```diff
@@ -0,0 +1,8 @@
+Destabilizer = ['+IZ', '+XI']
```

Comment:
Why "destabilizer" and not stabilizer?

Reply:
Basically just because its the first thing qiskit outputs